apps/entry/widgets/organigram_widget.py

```diff
@@ -22,8 +63,14 @@ def update_attribute(widget, data, widget_data):
             'data': {
                 'excel': {
                     'type': 'list',
-                    'value': [v.get('name') for v in selected_nodes],
+                    # TODO: Add migration for this fix
```

Comment:
Is the migration added already? If not, we can do that in this PR.

Reply:
Added.